### ISSUE TITLE
Footer text color

### ISSUE THREAD

**Issue by Uzmakh** [Anupkjha2601/food-recipes-website#217]
The footer text color is not visible enough on small screens.
I will like to make the background image so blurry that the text can be read
![food-10](https://github.com/Anupkjha2601/food-recipes-website/assets/91914613/6db68a6a-f600-4fd4-a0e5-293e44fef405)
.Here the color of the button also needs to be matched with the image colors.I will work on that also.

**Comment by ashutoshrana8118** [comment]
I can change the color of the button as per the image color if u assign me on this issue

**Comment by Uzmakh** [comment]
No,I am closing this issue as the button is getting its styling through theme.

**Comment by manoranjanonline** [comment]
please assign me this , i wants to contribute under GSSOC'23

**Comment by PIYUSH1525** [comment]
Hey, 
As the footer text color is not visible enough on small screens I want it to correct it and make it visible for small screens
